OGP1617-Asteroids/src/asteroids/model/Ship.java

```diff
@@ -66,6 +80,7 @@ public Ship(double x, double y, double xV, double yV, double orientation, double
 		this.setVelocity(xV, yV);
 		this.setOrientation(orientation);
 		this.radius = radius;
+		this.massDensity = massDensity;

```

Comment:
Je mag alleen final properties initialiseren in de constructor, voor massDensity best setMassDensity(massDensity)